packages/maskbook/src/plugins/Collectible/UI/ListingByPriceCard.tsx

```diff
@@ -102,7 +102,7 @@ export function ListingByPriceCard(props: ListingByPriceCardProps) {
                 accountAddress: account,
                 startAmount: Number.parseFloat(amount),
                 endAmount: endingPriceChecked && endingAmount ? Number.parseFloat(endingAmount) : undefined,
-                listingTime: futureTimeChecked ? toUnixTimestamp(scheduleDateTime) : undefined,
+                listingTime: futureTimeChecked ? toUnixTimestamp(dateTime) : undefined,
```

Comment:
Set `expirationTime` if `endingPriceChecked` is true.

Reply:
Fixed. And add an individual variable for expirationTime